plugins/connection/docker.py

```diff
@@ -268,6 +274,12 @@ def exec_command(self, cmd, in_data=None, sudoable=False):
         display.debug("done communicating")
 
         display.debug("done with docker.exec_command()")
+
+        # When running on Windows, stderr may contain CLIXML encoded output
+        if getattr(self._shell, "_IS_WINDOWS", False) and stderr.startswith(b"#< CLIXML"):
+            from ansible.plugins.shell.powershell import _parse_clixml
```

Comment:
I would be concerned with importing this function, it's currently a "private" function and the location has changed in the past before. You might be better off just copying the function here rather than trying to import this then find out a future Ansible version has changed it once again.

Reply:
@jborean93 copied `_parse_clixml` to `docker-py`